eclass/xdg-utils.eclass

```diff
@@ -86,21 +86,17 @@ xdg_icon_cache_update() {
 	fi
 
 	ebegin "Updating icons cache"
-	local dir f retval=0
-	local fails=( )
-	for dir in "${EROOT%/}"/usr/share/icons/*
-	do
-		if [[ -f "${dir}/index.theme" ]] ; then
-			local rv=0
-			gtk-update-icon-cache -qf "${dir}"
-			rv=$?
-			if [[ ! $rv -eq 0 ]] ; then
+	local dir retval=0
+	local fails=()
+	for dir in "${EROOT%/}"/usr/share/icons/*; do
+		if [[ -f ${dir}/index.theme ]]; then
+			if ! gtk-update-icon-cache -qf "${dir}"; then
 				debug-print "Updating cache failed on ${dir}"
 				# Add to the list of failures
 				fails+=( "${dir}" )
```

Comment:
For consistency, either do
```suggestion
				fails+=("${dir}")
```
here, or keep the blank in `local fails=( )` above.